### PR TITLE
add custom elements detection

### PR DESCRIPTION
**Pull request by lukasoppermann**
This is the first draft to address #2140.

However, I am at a loss as to how I should add test to test the detection. @ryanseddon could you please advise? Thanks.

### CODE REVIEW

File: feature-detects/custom-elements.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "name": "Specs for Custom Elements",
+    "href": "https://www.w3.org/TR/custom-elements/"
+  }]
+}
+!*/
+/* DOC
+Detects support for the Custom Elements API, to create custom html elements via js
+*/
+define(['Modernizr'], function(Modernizr) {
+  Modernizr.addTest('customelements', typeof window.customElements === 'undefined');
```
Comment (ryanseddon) [inline feature-detects/custom-elements.js]: `'customElements' in window` should be fine

**Comment by lukasoppermann** [comment]
Thanks @ryanseddon, changed it.

**Comment by ryanseddon** [comment]
> However, I am at a loss as to how I should add test to test the detection. @ryanseddon could you please advise?

What do you mean by this?

**Comment by ryanseddon** [comment]
I think you reverted your change to config-all.json too

**Comment by lukasoppermann** [comment]
Hey @ryanseddon, I never changed `config-all.json`, added a change now.

> > However, I am at a loss as to how I should add test to test the detection. @ryanseddon could you please advise?
> 
> What do you mean by this?

Well, I was wondering if there is a test to test, that the supportTest is correct?

**Comment by lukasoppermann** [comment]
Hey @ryanseddon can this be merged now?

**Pull request merged**